.gitignore

```diff
@@ -3,6 +3,9 @@
 build/
 __pycache__
 
+# Testa data files are not currently tracked in git
+testfiles*
```

Comment:
I'm not sure where this comes from. `old_test_dir` (extracted from the tarball testfiles) is one level up and shouldn't appear in the Git repo, `abs_test_dir` (created on the fly during tests) is in directories already ignored (or should be) like `build` and `.tox`.

Reply:
removed additiong to .gitirnore completely – build and .tox were already there